example.hpp

```diff
@@ -0,0 +1,117 @@
+/*
```

Comment:
Was this file accidentally commented out?

Reply:
yes